packages/typespec-vscode/README.md

```diff
@@ -35,3 +35,14 @@ This setting provides the ability to configure where the tsp compiler is located
   "typespec.tsp-server.path": "${workspaceFolder}/my-nested-project/node_modules/@typespec/compiler"
 }
 ```
+
+### `typespec.tsp.path`: Configure the tsp path
```

Comment:
This feels a bit unfortunate to have both this and server path when they should point to the same folder

Reply:
also as this technically take the input path I think the compiler should always be able to be resolved relative to it(Looking up)

Reply:
what do you mean by the "input" path here? Another way I can think about is to figure out the cli path from the tsp-server path. i.e. replace tsp-server with tsp or cli properly (ideally, they should be together, right?). It just feels a little strange that cli path is actually defined by a server's setting. But I can live with it. how do you think about it?

Reply:
if you feel like we need this property I would deprecate the old `tsp-server` one and have this one resolve both.

But yeah I assume this means you can compile/watch the current file? If so you should be able to walk up until you see the compiler in a parent node_modules folder

Reply:
i am fine with auto-resolve. code updated. thx